.ado/templates/apple-xcode-build-static-libs.yml

```diff
@@ -62,7 +62,7 @@ steps:
       xcode_actions: 'build'
       xcode_scheme: 'FluentTester'
       xcode_configuration: 'Debug'
-      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides.xcconfig'
+      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides.xcconfig' ONLY_ACTIVE_ARCH=NO
```

Comment:
So... this totally works, but It would be cleaner to not need to manually add the xcconfig here. I'll follow up with the react-native-test-app team on getting the change integrated upstream